shaders/src/getters.vs

```diff
@@ -142,6 +142,9 @@ vec4 computeWorldPosition() {
 #else
     mat4 transform = getWorldFromClipMatrix();
     vec4 position = transform * getPosition();
+    if (abs(position.w) < MEDIUMP_FLT_MIN) {
+        position.w = sign(position.w) != 0.0 ? sign(position.w) * MEDIUMP_FLT_MIN : MEDIUMP_FLT_MIN;
```

Comment:
Why the test here? If you're going to call sign() every time, might as well do sign(w) * FLT_MIN as the expression. Even better just write:

position.w = position.w < 0.0 ? -FLT_MIN : FLT_MIN